setup-issue-templates/draft-schedule.md

```diff
@@ -0,0 +1,4 @@
+Update `workshop/SCHEDULE.md` to point to appropriate materials for your workshop.
+
+  - Add relative links to PDFs in `slides` to the schedule.
+  - We recommend using [htmlpreview](https://github.com/htmlpreview/htmlpreview.github.com) to display rendered versions of R Notebooks that are tied to a specific release in the [`AlexsLemonade/training-modules`](https://github.com/AlexsLemonade/training-modules). You can use the `{{site.release_tag}}` convention used throughout the template.
```

Comment:
In the draft schedule, we might not want the tags? Or should we note that links might not work if you do?

Reply:
The way this is currently set up is that it `{{site.release_tag}}` would point to `master`:

https://github.com/AlexsLemonade/training-specific-template/blob/c12a4d7a64830a2a02532eafbd6233e0eab86cf0/_config.yaml#L13

So we would expect the links to work and then when you finalize the schedule hopefully it would be using the correct release tag. The idea being that you can stub in any notebooks where the names are unlikely to change and then you're ready to go come finalization time.